data/json/recipes/tools/tools_hand.json

```diff
@@ -1163,5 +1163,20 @@
     "autolearn": true,
     "tools": [ [ [ "cordless_drill", 20 ] ] ],
     "components": [ [ [ "sheet_metal_small", 1 ] ] ]
-  }
+  },
+  {
+   "type": "recipe",
+   "activity_level": "LIGHT_EXERCISE",
+   "result": "casting_mold",
+   "category": "CC_OTHER",
+   "subcategory": "CSC_OTHER_TOOLS",
+   "skill_used": "fabrication",
+   "difficulty": 3,
+   "time": "15 m",
+   "autolearn": true,
+   "components": [
+     [ [ "material_sand", 5 ]],
+     [ [ "2x4", 8 ] ] 
```

Comment:
Would this need nails to get the 2x4s to stay together?

Reply:
Ah, yes right! I suppose glue would work too.